src/box/lua/ctl.c

```diff
@@ -185,6 +187,26 @@ lbox_ctl_wal_sync(struct lua_State *L)
 	return 0;
 }
 
+static int
+lbox_ctl_deactivate_replica(struct lua_State *L)
+{
+	const char *uuid_str = luaT_checkstring(L, 1);
+	struct tt_uuid uuid;
+	if (tt_uuid_from_string(uuid_str, &uuid) != 0)
+		luaT_error(L);
```

Comment:
`tt_uuid_from_string` doesn't set diag. Please use `IllegalParams` error and add a test for invalid arguments.

Reply:
Thanks! Fixed and added tests with invalid UUID.